Resources/Prototypes/Entities/Structures/Windows/reinforced.yml

```diff
@@ -62,6 +62,22 @@
     minPressureDelta: 10000
     scalingType: Threshold
 
+ # Start DeltaV additions
+- type: entity
+  id: ReinforcedTintedWindow
+  name: reinforced tinted window
+  parent: ReinforcedWindow
+  components:
+  - type: Sprite
+    drawdepth: WallTops
+    sprite: _DV/Structures/Windows/reinforced_tinted_window.rsi
+  - type: Icon
+    sprite: _DV/Structures/Windows/reinforced_tinted_window.rsi
+  - type: IconSmooth
+    base: rtwindow
+  - type: Occluder
+  # End DeltaV additions
```

Comment:
Move to a new file under the _DV namespace.

I believe there's already one for tinted windows in general (see `Resources/Prototypes/_DV/Entities/Structures/Windows/tinted_windows.yml`)